apps/test-app/frontend/src/components/tree-widget/stateless-v2/Types.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { PresentationInstanceFilterInfo } from "@itwin/presentation-components";
+
+/** @beta */
+export type InfoNodeTypes = "ResultSetTooLarge" | "Placeholder" | "Unknown";
```

Comment:
It's not entirely clear what `Placeholder` means. Maybe rename to `ChildrenPlaceholder`?